docs/MIGRATING_V1.md

```diff
@@ -0,0 +1,316 @@
+# Upgrading to Project Factory v1.0
+
+The v1.0 release of Project Factory is a backwards incompatible release and
+features significant changes, specifically with how GSuite resources are
+managed. A state migration script is provided to update your existing states
+to minimize or eliminate resource re-creations. 
+
+NOTE: This document is a work in progress as v1.0 has not yet been released.
```

Comment:
Can this be dropped assuming we just wait for the 1.0 release to merge this?

Reply:
Yes, that note was specifically for this pull request review and not the final document.

Reply:
Comment dropped.